### ISSUE TITLE
No diagnostic given for invalid call

### ISSUE THREAD

**Issue by schweitzpgi** [flang-compiler/f18-llvm-project#1043]
```fortran
 real, pointer :: x
 call extern1(null(x))
end
```

**Comment by sscalpone** [comment]
Fixed upstream:
```
error: Semantic errors in b1043.f90
./b1043.f90:2:2: error: Null pointer argument requires an explicit interface
   call extern1(null(x))
   ^^^^^^^^^^^^^^^^^^^^^
```
